src/lib/connection.ts

```diff
@@ -94,6 +94,8 @@ export interface Connection {
     controller: string,
     name: string
   ) => Promise<WriteQueryResult>;
+  getController: (tableName: string) => Promise<string>;
```

Comment:
Just wondering out loud... should we namespace all of these under controller? so controller.get, controller.set, controller.lock?

Reply:
That would work.  My tendency is usually to try to keep the api intuitive to a developer when possible. Currently the Contract functions, RPC endpoints, and sdk methods all match.  This makes sense to me, but maybe a `controller` object with functions on it would be better?
If we do switch, i think we would want to make sure to bind the context of the functions to "tableland", e.g. inside of tableland.controller.set we would want `this` to be tableland not controller so we can call `this.checkNetwork` etc...  Alternatively we could abstract the idea of a `super` as `this.super` (or something like that) that points to tableland, but that seems more complicated.